src/__tests__/Schema.test.ts

```diff
@@ -376,3 +378,767 @@ describe('transformPathsToDbPositions', () => {
 		expect(schema.transformPathsToDbPositions(['not.here'])).toEqual([]);
 	});
 });
+
+describe('utility types', () => {
```

Comment:
I don't want to make a bunch of tests because these aren't real tests. They're just a container to keep the type assertions in.

I did add comments to each individual test section.